src/app/coursegrab/models/session.py

```diff
@@ -10,6 +10,7 @@ class Session(db.Model):
     device_token = db.Column(db.String, nullable=True)
     device_type = db.Column(db.String, nullable=True)
 
+    last_used = db.Column(db.DateTime, nullable = False)
```

Comment:
lol how come this one has `nullable = False` but the other ones don't have spaces hahaha